static/app/views/replays/types.tsx

```diff
@@ -181,6 +181,17 @@ export type DeadRageSelectorListResponse = {
   data: {count_dead_clicks: number; count_rage_clicks: number; dom_element: string}[];
```

Comment:
does this type also include `element: ReplayClickElement` in there?

Reply:
yes!

Reply:
forgot to update this type along the way